src/Analyzers/CSharp/Analyzers/ConvertSwitchStatementToExpression/ConvertSwitchStatementToExpressionDiagnosticAnalyzer.cs

```diff
@@ -90,9 +97,6 @@ private void AnalyzeSyntax(SyntaxNodeAnalysisContext context)
                     .Add(Constants.ShouldRemoveNextStatementKey, shouldRemoveNextStatement.ToString(CultureInfo.InvariantCulture))));
         }
 
-        public override bool OpenFileOnly(OptionSet options)
-            => false;
```

Comment:
is this a behavioral change?

Reply:
No, the base type always defaults OpenFileOnly to false.